### DISCUSSION TITLE
Add support for TOC on asciidoc

### DISCUSSION THREAD

**Discussion by stian90** [colinwilson/lotusdocs#165] [Feature Requests]
I´m using lotusdocs with our documentation that is written in asciidoc. The pages work fine, but we are missing "toc" on every document/page. From my understanding hugo uses asciidoctor to convert the pages from asciidoc to html before showing them of. 

Is this a future that is possible to add at some point?

**Comment by fofwisdom** [comment]
I use AsciiDoc, too. I’ve customized both Lotus Docs and Hugo to suit my purposes.
* https://docs.netspheres.org/
* https://github.com/fofwisdom/lotusdocs-asciidoc
